### PR TITLE
Remove polyfills

### PR DESCRIPTION
**Pull request by pzac**

### CODE REVIEW

File: lib/hawk.rb [hunk 1]
```diff
@@ -1,8 +1,11 @@
 ##
 # Hawk entry point.
 #
+#
+
+require 'active_support/core_ext/array/wrap'
```
Comment (vjt) [inline lib/hawk.rb]: Is this enough? Or, why is it needed? We are using `pluralize` but not requiring it explicitly.

Comment (pzac) [inline lib/hawk.rb]: I know, that's strange, but without this explicit `require` I get a
`#<NoMethodError: undefined method 'wrap' for Array:Class>`. I have the feeling it has to do with ActiveSupport's `require`-ing and autoloading.

Comment (vjt) [inline lib/hawk.rb]: OK, cool, thanks.

**Comment by vjt** [comment]
Oh, look how a bit of Not Invented Here syndrome goes away.

@jblackman no more refinements in Hawk, you can start testing it on Ruby 1.9 :-)

**Pull request merged**